client/ts/src/client.ts

```diff
@@ -176,12 +178,15 @@ export class ManifestClient {
           owner: payerKeypair.publicKey,
           wrapperState: wrapperKeypair.publicKey,
         });
-      const claimSeatIx: TransactionInstruction = createClaimSeatInstruction({
-        manifestProgram: MANIFEST_PROGRAM_ID,
-        owner: payerKeypair.publicKey,
-        market: marketPk,
-        wrapperState: wrapperKeypair.publicKey,
-      });
+      const claimSeatIx: TransactionInstruction = createClaimSeatInstruction(
```

Comment:
i think you need to call the ui wrapper or bot wrapper version of this fn depending on program because they require different accounts. currently, all of the client is connected to bot wrapper

https://github.com/CKS-Systems/manifest/blob/main/programs/ui-wrapper/src/instruction.rs#L27
https://github.com/CKS-Systems/manifest/blob/main/programs/wrapper/src/instruction.rs#L23
https://github.com/CKS-Systems/manifest/blob/e3b726ee6a937e3f5431b8e457f5d2595facdf67/client/ts/src/client.ts#L37